wait.go

```diff
@@ -33,7 +33,7 @@ func (confirm confirmFunc) wrap(wait waitFunc) waitFunc {
 	}
 }
 
-func (d *App) WaitFunc(sv *Service, confirm confirmFunc) (waitFunc, error) {
+func (d *App) WaitFunc(sv *Service, confirm confirmFunc, waitUntilType *string) (waitFunc, error) {
```

Comment:
```suggestion
func (d *App) WaitFunc(sv *Service, confirm confirmFunc, waitUntilType string) (waitFunc, error) {
```

`waitUntilType` is not needed as a pointer type. Using `string` works fine.

Reply:
Thanks!  I've stopped using pointers!